pkg/security/ebpf/c/dentry.h

```diff
@@ -271,10 +273,11 @@ static __attribute__((always_inline)) int resolve_dentry(struct dentry *dentry,
     }
 
     // If the last next_id isn't null, this means that there are still other parents to fetch.
-    // TODO: use BPF_PROG_ARRAY to recursively fetch 32 more times. For now, add a fake parent to notify
-    // that we couldn't fetch everything.
+    // TODO: use BPF_PROG_ARRAY to recursively fetch 32 more times.
 
-    map_value.name[0] = 0;
+    // 21 is the ASCII code for NAK = Negative Acknowledge. This value is used to notify that we couldn't resolve the
+    // entire path.
+    map_value.name[0] = 0x15;
```

Comment:
We need a custom value to identify that we reached the limits of the agent. Using `/` would not work since the paths naturally end with `/` 😢  I guess we could also count in user space the depth of the path instead of looking for a specific value ...